packages/loader/driver-utils/package.json

```diff
@@ -11,17 +11,29 @@
 	"license": "MIT",
 	"author": "Microsoft and contributors",
 	"sideEffects": false,
-	"main": "dist/index.js",
-	"module": "lib/index.js",
+	"exports": {
```

Comment:
Probably best to remove the exports field from packages since the recent release had issues related to it. Making those changes separately is probably best.